src/main/java/uk/gov/pay/api/model/search/directdebit/DDSearchResponse.java

```diff
@@ -4,13 +4,13 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.pay.api.model.links.SearchNavigationLinks;
-import uk.gov.pay.api.model.search.ISearchPagination;
+import uk.gov.pay.api.model.search.SearchPagination;
 
 import java.util.List;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class DDSearchResponse implements ISearchPagination<SearchNavigationLinks> {
+public class DDSearchResponse implements SearchPagination {
```

Comment:
Would you like to rename `DD` to `DirectDebit`? May be check other team members

Reply:
It seems we have mostly DD for direct debit related things in publicapi, except `SearchDirectDebitPayments` 😬

Reply:
I will rename everything to DirectDebit, ha